### PR TITLE
Add types for bcswipe

### PR DESCRIPTION
**Pull request by stof**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

I haven't use `dts-gen --dt` because this package defines a jQuery plugin, which cannot be detected by dts-gen

### CODE REVIEW

File: types/bcswipe/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// Type definitions for bcswipe 2.0
+// Project: https://github.com/briggySmalls/bcSwipe
+// Definitions by: Christophe Coevoet <https://github.com/stof>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.7
```
Comment (peterblazejewicz) [inline types/bcswipe/index.d.ts:5]:
```suggestion
```
https://github.com/DefinitelyTyped/DefinitelyTyped#support-window

Comment (stof) [reply types/bcswipe/index.d.ts]: @peterblazejewicz done

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@stof thx!

**Comment by stof** [comment]
Ready to merge

**Pull request merged**